manage-data/lifecycle/data-tiers.md

```diff
@@ -2,31 +2,447 @@
+
+
+### Cold tier [cold-tier]
+
+When you no longer need to search time series data regularly, it can move from the warm tier to the cold tier. While still searchable, this tier is typically optimized for lower storage costs rather than search speed.
+
+For better storage savings, you can keep [fully mounted indices](/deploy-manage/tools/snapshot-and-restore/searchable-snapshots.md#fully-mounted) of [{{search-snaps}}](https://www.elastic.co/guide/en/elasticsearch/reference/current/ilm-searchable-snapshot.html) on the cold tier. Unlike regular indices, these fully mounted indices don’t require replicas for reliability. In the event of a failure, they can recover data from the underlying snapshot instead. This potentially halves the local storage needed for the data. A snapshot repository is required to use fully mounted indices in the cold tier. Fully mounted indices are read-only.
+
+Alternatively, you can use the cold tier to store regular indices with replicas instead of using {{search-snaps}}. This lets you store older data on less expensive hardware but doesn’t reduce required disk space compared to the warm tier.
+
+
+### Frozen tier [frozen-tier]
+
+Once data is no longer being queried, or being queried rarely, it may move from the cold tier to the frozen tier where it stays for the rest of its life.
+
+The frozen tier requires a snapshot repository. The frozen tier uses [partially mounted indices](/deploy-manage/tools/snapshot-and-restore/searchable-snapshots.md#partially-mounted) to store and load data from a snapshot repository. This reduces local storage and operating costs while still letting you search frozen data. Because {{es}} must sometimes fetch frozen data from the snapshot repository, searches on the frozen tier are typically slower than on the cold tier.
+
+## Configure data tiers [configure-data-tiers]
+
+Follow the instructions for your deployment type to configure data tiers.
+
+### {{ess}} or {{ece}} [configure-data-tiers-cloud]
+
+The default configuration for an {{ecloud}} deployment includes a shared tier for hot and content data. This tier is required and can’t be removed.
+
+#### Add a data tier
+
+To add a warm, cold, or frozen tier when you create a deployment:
+
+1. On the **Create deployment** page, click **Advanced Settings**.
+2. Click **+ Add capacity** for any data tiers to add.
+3. Click **Create deployment** at the bottom of the page to save your changes.
+
+:::{image} ../../images/elasticsearch-reference-ess-advanced-config-data-tiers.png
+:alt: {{ecloud}}'s deployment Advanced configuration page
+:class: screenshot
+:::
+
+To add a data tier to an existing deployment:
+
+1. Log in to the [{{ecloud}} console](https://cloud.elastic.co?page=docs&placement=docs-body).
```

Comment:
```suggestion
1. Log in to the [{{ecloud}} console](https://cloud.elastic.co?page=docs&placement=docs-body) or the {{ece}} admin console.
```

ECE users have a different log in location. The term "Elastic Cloud" also probably needs to change, but I think that's a problem for another day. :-)

Reply:
I'm going to use tabs for this!

Reply:
Oh wait never mind! I will use your suggestion.